docs/layers/lightlayer.rst

```diff
@@ -47,6 +47,9 @@ LightLayer.lightType
 
 ``app.project.item(index).layer(index).lightType``
 
+.. note::
+   ``LightType.ENVIRONMENT`` was added in After Effects (Beta) 24.3.0 (Build 36)
```

Comment:
This should be only the version when it was added to release (e.g. After Effects 24.3).